yt/frontends/stream/tests/test_stream_unstructured.py

```diff
@@ -8,31 +8,17 @@ def test_multi_mesh():
         [[0.0, 0.0], [1.0, 0.0], [1.0, 1.0], [0.0, 1.0]], dtype=np.float64
     )
 
-    connect1 = np.array(
-        [
-            [0, 1, 3],
-        ],
-        dtype=np.int64,
-    )
-    connect2 = np.array(
-        [
-            [1, 2, 3],
-        ],
-        dtype=np.int64,
-    )
+    connect1 = np.array([[0, 1, 3]], dtype=np.int64)
```

Comment:
@jcoughlin11 hey, any idea why this got changed?  Did black get run differently?